### PR TITLE
Sit

### PR DESCRIPTION
**Pull request by RebeccaStankus**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,3 +41,5 @@
 ehthumbs.db
 Thumbs.db
 
+dbInfo.json
```
Comment (zfox23) [inline .gitignore]: why is this here?

Comment (RebeccaStankus) [reply .gitignore]: I added it to my git ignore because it somehow got removed
